webapp/src/components/systemConsole/surveyDateTime/surveyDateTime.tsx

```diff
@@ -2,64 +2,76 @@
 // See LICENSE.txt for license information.
 
 import {format, parse} from 'date-fns';
-import React, {useEffect, useState} from 'react';
+import React, {useCallback, useEffect, useMemo, useState} from 'react';
 
 import SurveyDateSelector from 'components/surveyDateSelector/surveyDateSelector';
 import SurveyTimeSelector from 'components/surveyTimeSelector/surveyTimeSelector';
+import type {CustomSettingChildComponentProp} from 'components/systemConsole/index';
 
-import type {CustomComponentProps} from 'types/mattermost-webapp';
+import type {DateTimeConfig} from 'types/mattermost-webapp';
 
 import './style.scss';
 
-const defaultSurveyTime = '09:00';
+function SurveyDateTime({id, setSaveNeeded, onChange, config, setInitialSetting}: CustomSettingChildComponentProp) {
+    // Default settings
+    const defaultSurveyTime = '09:00';
```

Comment:
Same comment as Daniel has for constants, they should be defined outside the function.

Reply:
Done.